docs/generated/redact_safe.md

```diff
@@ -10,46 +10,47 @@ pkg/kv/kvserver/raft.go | `SnapshotRequest_Type`
 pkg/roachpb/data.go | `LeaseSequence`
 pkg/roachpb/data.go | `ReplicaChangeType`
 pkg/roachpb/metadata.go | `NodeID`
-pkg/roachpb/metadata.go | `StoreID`
+pkg/roachpb/metadata.go | `RangeGeneration`
 pkg/roachpb/metadata.go | `RangeID`
 pkg/roachpb/metadata.go | `ReplicaID`
-pkg/roachpb/metadata.go | `RangeGeneration`
 pkg/roachpb/metadata.go | `ReplicaType`
+pkg/roachpb/metadata.go | `StoreID`
 pkg/roachpb/method.go | `Method`
-pkg/sql/catalog/descpb/structured.go | `ID`
-pkg/sql/catalog/descpb/structured.go | `FamilyID`
-pkg/sql/catalog/descpb/structured.go | `IndexID`
-pkg/sql/catalog/descpb/structured.go | `DescriptorVersion`
-pkg/sql/catalog/descpb/structured.go | `IndexDescriptorVersion`
 pkg/sql/catalog/descpb/structured.go | `ColumnID`
-pkg/sql/catalog/descpb/structured.go | `MutationID`
+pkg/sql/catalog/descpb/structured.go | `ConstraintType`
 pkg/sql/catalog/descpb/structured.go | `ConstraintValidity`
 pkg/sql/catalog/descpb/structured.go | `DescriptorMutation_Direction`
 pkg/sql/catalog/descpb/structured.go | `DescriptorMutation_State`
 pkg/sql/catalog/descpb/structured.go | `DescriptorState`
-pkg/sql/catalog/descpb/structured.go | `ConstraintType`
-pkg/sql/sem/tree/table_ref.go | `ID`
+pkg/sql/catalog/descpb/structured.go | `DescriptorVersion`
+pkg/sql/catalog/descpb/structured.go | `FamilyID`
+pkg/sql/catalog/descpb/structured.go | `ID`
+pkg/sql/catalog/descpb/structured.go | `IndexDescriptorVersion`
+pkg/sql/catalog/descpb/structured.go | `IndexID`
+pkg/sql/catalog/descpb/structured.go | `MutationID`
 pkg/sql/sem/tree/table_ref.go | `ColumnID`
-pkg/storage/enginepb/mvcc3.go | `MVCCStatsDelta`
+pkg/sql/sem/tree/table_ref.go | `ID`
 pkg/storage/enginepb/mvcc3.go | `*MVCCStats`
-pkg/util/hlc/timestamp.go | `Timestamp`
+pkg/storage/enginepb/mvcc3.go | `MVCCStatsDelta`
 pkg/util/hlc/timestamp.go | `ClockTimestamp`
-pkg/util/log/redact.go | `reflect.TypeOf(true)`
+pkg/util/hlc/timestamp.go | `Timestamp`
+vendor/github.com/cockroachdb/pebble/internal/humanize/humanize.go | `FormattedString`
```

Comment:
Shouldn't this be sorted at the end?

Reply:
It is! It's at the end of the types that implement `SafeValue`.

The line afterwards is a different block, with the list of calls to `RegisterSafeType`.

Reply:
Ah, LGTM.